platform-core/src/main/kotlin/org/dashevo/platform/Names.kt

```diff
@@ -232,11 +232,12 @@ class Names(val platform: Platform) {
      * @param startAtIndex Int
      * @return List<Documents>
      */
-    fun search(text: String, parentDomain: String, retrieveAll: Boolean, startAtIndex: Int = 0): List<Document> {
+    fun search(text: String, parentDomain: String, retrieveAll: Boolean, limit: Int = -1, startAtIndex: Int = 0): List<Document> {
```

Comment:
Nice addition.

Reply:
Yes, in this case, I was taking a shortcut instead of pulling in your PR:  #30 - as it turns out my version has a different order of arguments.